--- origsrc/rbwebkitgtk/extconf.rb	2008-06-18 00:08:49.000000000 -0500
+++ src/rbwebkitgtk/extconf.rb	2008-06-18 00:13:08.343750000 -0500
@@ -51,11 +51,11 @@
 #   exit 1
 # end
 
-#add_depend_package("glib2", "glib/src", TOPDIR)
-#add_depend_package("gtk2", "gtk/src", TOPDIR)
-
 have_library("webkit-1.0") or exit 1
 
+add_depend_package("glib2", "glib/src", TOPDIR)
+add_depend_package("gtk2", "gtk/src", TOPDIR)
+
 # have_func('gtk_moz_embed_new') or exit 1
 # have_func('gtk_moz_embed_set_profile_path') or exit 1
 
--- origsrc/rbwebkitgtk/sample/ruby-webkit-browser.rb	2008-06-18 00:08:49.000000000 -0500
+++ src/rbwebkitgtk/sample/ruby-webkit-browser.rb	2008-06-18 00:14:17.140625000 -0500
@@ -1,5 +1,5 @@
 
-require 'src/lib/webkit'
+require 'webkit'
 
 win = Gtk::Window.new
 win.signal_connect("destroy") { Gtk.main_quit }
